virttest/qemu_vm.py

```diff
@@ -1690,13 +1699,6 @@ def add_secure_guest_descriptor(params):
                     cmdline=process_sandbox(
                         devices,
                         "add")))
-        elif qemu_sandbox == "off":
-            devices.insert(
-                StrDev(
-                    'qemu_sandbox',
-                    cmdline=process_sandbox(
-                        devices,
-                        "rem")))
```

Comment:
I think we have to keep this clause for the compatibility of the param, as the explicit expression of the `off` value would still be a valid scenario of testing.

Reply:
Updated now, thanks!